BLUESPAWN-client/src/util/log/CLISink.cpp

```diff
@@ -11,7 +11,10 @@ namespace Log {
 		SetConsoleTextAttribute(hConsole, static_cast<WORD>(color));
 	}
 
+	CLISink::CLISink(const HandleWrapper& hMutex) : hMutex{ hMutex } {}
+
 	void CLISink::LogMessage(const LogLevel& level, const std::string& message, const std::optional<HuntInfo> info, const std::vector<std::shared_ptr<DETECTION>>& detections){
+		auto mutex = AcquireMutex(hMutex);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'mutex' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5011005)